src/JetReconstruction.jl

```diff
@@ -89,4 +89,8 @@ export jetsplot, animatereco
 include("JSONresults.jl")
 export FinalJet, FinalJets
 
+# Substructure modules
+include("Substructure.jl")
```

Comment:
Just for internal ordering, this should be after the `include("GenericAlgo.jl")` at L70, coming before the utility parts of the package.